frontend-web/webclient/app/Applications/Jobs/View.tsx

```diff
@@ -481,12 +492,17 @@ const InQueueText: React.FunctionComponent<{job: Job}> = ({job}) => {
 
     useEffect(() => {
         const support = job.status.resolvedSupport?.support as ComputeSupport;
-        if (support.docker.utilization === true || support.virtualMachine.utilization === true) {
+        const appType = getAppType(job);
+        if (
+            (appType === "DOCKER" && support.docker.utilization === true) ||
+            (appType === "VIRTUAL_MACHINE" && support.virtualMachine.utilization === true)
+        ) {
             setUtilization(compute.jobs.retrieveUtilization({jobId: job.id}))
         }
     }, [job]);
 
     return <>
+        {/* Note(Jonas): Line below is outdated, right? */}
         <Heading.h2>{CONF.PRODUCT_NAME} is preparing your job</Heading.h2>
```

Comment:
Should this be based on the provider instead?

Reply:
Yes, I think it should say something different. This is probably the simplest solution.

```suggestion
        <Heading.h2>Your job is being prepared</Heading.h2>
```